helm/content-collection-unfolder/app-configs/content-collection-unfolder_delivery.yaml

```diff
@@ -8,4 +8,6 @@ env:
   WRITER_HEALTH_URI: http://content-collection-rw-neo4j:8080/__health
   CONTENT_RESOLVER_URI: http://document-store-api:8080/content/
   CONTENT_RESOLVER_HEALTH_URI: http://document-store-api:8080/__health
+  RELATIONS_RESOLVER_URI: http://relations-api/contentcollection/{uuid}/relations
+  RELATIONS_RESOLVER_HEALTH_URI: http://relations-api/__health
```

Comment:
You should access the relations-api on the 8080 port, like http://relations-api:8080/..